tests/opbanning/test_op_banning.py

```diff
@@ -32,6 +33,19 @@
     "SELFDESTRUCT",
 ]
 
+banned_opcodes_for_undeployed = [
+    "EXTCODESIZE",
```

Comment:
this is a rule: EXTCODESIZE_to_undeployed

Reply:
Addressed.